client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1991,6 +1991,21 @@ Undocumented.prototype.getGuidedTransferStatus = function( siteId ) {
 };
 
 /**
+ * Requests the status of a guided transfer
+ *
+ * @param {int} siteId  The site ID
+ * @returns {Promise} Resolves to the response containing the transfer status
+ */
+Undocumented.prototype.saveGuidedTransferHostDetails = function( siteId, hostDetails ) {
```

Comment:
Hmm, I just looked inside that `client/lib/wpcom-undocumented/lib/export.js` file and I don't recognize the code 😅  . That looks like a relic from a previous iteration of export. All the currently used API funcs are under `wpcom-undocumented/lib/undocumented.js`.

I agree it would be nice to have some more organization there, `undocumented.js` is a monster at ~2000 lines.

In this case, I think number 2 makes sense - the `site` lib. I'll refactor that and make a note to clean up the unused export stuff in a separate PR.